shopsafe-backend/src/main/java/com/google/sps/QueryCovidStats.java

```diff
@@ -0,0 +1,188 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+import java.util.UUID;
+import java.lang.Long;
+
+
+/*
+ * Class for using BigQuery to query the COVID-19 public forecast dataset.
+ */
+public class QueryCovidStats {
+
+    private long cases;
+    private long deaths;
+    private long activeCases;
+    private boolean failedQuery;
+
+    private QueryCovidStats(long cases, long deaths, long activeCases, boolean failedQuery) {
+        this.cases = cases;
+        this.deaths = deaths;
+        this.activeCases = activeCases;
+        this.failedQuery = failedQuery;
+    }
+
+    public long getCases() {
+        return cases;
+    }
+
+    public long getDeaths() {
+        return deaths;
+    }
+
+    public long getActiveCases() {
+        return activeCases;
+    }
+
+    public boolean failedQuery() {
+        return failedQuery;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        // If the object is compared with itself then return true   
+        if (o == this) { 
+            return true; 
+        } 
+  
+        /* Check if o is an instance of QueryNYT or not 
+          "null instanceof [type]" also returns false */
+        if (!(o instanceof QueryCovidStats)) { 
+            return false; 
+        } 
+          
+        // typecast o to QueryNYT so that we can compare data members  
+        QueryCovidStats other = (QueryCovidStats) o; 
+
+        return (this.cases == other.cases &&
+            this.deaths == other.deaths &&
+            this.activeCases == other.activeCases &&
+            this.failedQuery == other.failedQuery
+            );
+    }
+    
+    /*
+     * Return instance of QueryCovidStats if possible, otherwise, return failure.
+     */
+    public static QueryCovidStats getCovidStatsFips(String fips) {
+
+        // Set up BigQuery service.
+        BigQuery bigquery = BigQueryOptions
+            .newBuilder().setProjectId("shopsafe-step-2020").build().getService();
+
+        // Prepare SQL query for getting cases and deaths from NYT.
+        QueryJobConfiguration queryConfigNYT = QueryJobConfiguration.newBuilder(
+            "SELECT confirmed_cases, deaths, date FROM `bigquery-public-data.covid19_nyt.us_counties` WHERE county_fips_code = '" +
+            fips +
+            "' ORDER BY date DESC LIMIT 1"
+            ).setUseLegacySql(false).build();
```

Comment:
There are some cases where a row doesn't include the fips code
![Screenshot 2020-07-09 at 9 51 59 AM](https://user-images.githubusercontent.com/39624330/87048651-fb7bc000-c1c9-11ea-8c53-dd9f0daefa75.png)

Reply:
It actually works fine. The reason that Kansas City doesn't have a fips code is because its not a county. It looks like they just put it in the county column as a hacky work around.

Reply:
gotcha, ok everything else looks good

Reply:
Just to make sure it works fine, I tried using downtown Kansas City and it worked. For the county, it returns Wyandotte.